pkg/workloads/console/acceptance/acceptance.go

```diff
@@ -125,7 +162,7 @@ func (r *Runner) Run(logger kitlog.Logger, config *rest.Config) {
 				console, err = client.WorkloadsV1Alpha1().Consoles(namespace).Get(consoleName, metav1.GetOptions{})
 				Expect(err).NotTo(HaveOccurred(), "could not find console")
 				return console.Status.Phase
-			}, 8*time.Second).Should(Equal(workloadsv1alpha1.ConsoleStopped))
```

Comment:
Note sure we actually get value from this. I had a test flake locally because of this timeout. It should normally be fast but worse case we wait a minute and then fail. If we are hung here, locally I find it useful to poke around the test cluster to see what is going on. If we hang in CI waiting a little longer doesn't seem a big deal to me.

Reply:
Yeah let's bump this up, I think that makes sense and shouldn't impact anything else.